Makefile_windows

```diff
@@ -367,20 +274,6 @@ contribs/gmf/fonts/fontawesome-webfont.%: node_modules/font-awesome/fonts/fontaw
 		--var examples=true \
 		--var source_map=.build/examples/$*.js.map $< > $@
 
-.build/ngeo.json: buildtools/mako_build.json .build/python-venv/Scripts/mako-render
-	PYTHONIOENCODING=UTF-8 .build/python-venv/Scripts/mako-render \
-		$(CLOSURE_VARS) \
-		--var lib=true \
-		--var src_set=ngeo \
-		--var source_map=dist/ngeo.js.map $< > $@
-
-.build/gmf.json: buildtools/mako_build.json .build/python-venv/Scripts/mako-render
-	PYTHONIOENCODING=UTF-8 .build/python-venv/Scripts/mako-render \
-		$(CLOSURE_VARS) \
-		--var lib=true \
-		--var src_set=contribs_gmf \
-		--var source_map=dist/gmf.js.map $< > $@
-
```

Comment:
@sbrunner, I removed these two blocks as well, because the use some files (`dist/gmf.js.map`, `dist/ngeo.js.map`) which are in the dist folder (which is not generated when using the Windows Makefile. I hope this is correct...?